packages/ethereum-contracts/contracts/CID_External.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import "./CID.sol";
+
+contract CID_External {
```

Comment:
To be honest I would rather external devs ignore this file, the only reason why it needs to exist is that hardhat doesn't have a way to directly import a library and call its functions, the library has to first be imported by a contract. I think at the very least `CID_Example` is a less confusing name than `CID_External` so I am happy to rename it to that. Also it might be a good idea to put it in a separate directory like `.../contracts/test`.

Reply:
What about `CID_Fixture.sol`?